demo/src/main/java/com/basgeekball/awesomevalidation/demo/DemoActivity.java

```diff
@@ -168,6 +148,22 @@ private void addValidationForEditText(Activity activity) {
         mAwesomeValidation.addValidation(activity, R.id.edt_zipcode, "\\d+", R.string.err_zipcode);
         mAwesomeValidation.addValidation(activity, R.id.edt_year, Range.closed(1900, Calendar.getInstance().get(Calendar.YEAR)), R.string.err_year);
         mAwesomeValidation.addValidation(activity, R.id.edt_height, Range.closed(0.0f, 2.72f), R.string.err_height);
+        mAwesomeValidation.addValidation(activity, R.id.edt_birth, new CustomValidation() {
+            @Override
+            public boolean compare(String input) {
+                try {
+                    Date date = new SimpleDateFormat("dd/MM/yyyy", Locale.US).parse(input);
+                    Calendar calendar = Calendar.getInstance();
+                    if (calendar.getTime().after(date)) //You can't be born in future!
+                        return true;
+                } catch (Exception e) {
+                    e.printStackTrace();
+                    return false;
```

Comment:
No need to return `false` here, it will fall to the final `return false` anyway.